spec/index.bs

```diff
@@ -11868,9 +11893,9 @@ dictionary GPURenderPassColorAttachment {
 
     : <dfn>resolveTarget</dfn>
     ::
-        A {{GPUTextureView}} describing the texture [=subresource=] that will receive the resolved
-        output for this color attachment if {{GPURenderPassColorAttachment/view}} is
-        multisampled.
+        Describes the texture [=subresource=] that will receive the resolved output for this color
+        attachment if {{GPURenderPassColorAttachment/view}} is multisampled.
+        The [=subresource=] is determined by calling [$resolve texture view$]({{GPURenderPassColorAttachment/resolveTarget}}).
```

Comment:
nit: "Resolve" is an overused term, especially here where it's right next to resolveTarget. Use some other wording like maybe "get view for texture binding"

Reply:
Apologies I just realized "get view for texture binding" doesn't make sense because this is not just for bindings. (I didn't review the whole PR.)

My best suggestion is "get texture view" or "get as texture view".